### PR TITLE
Addedd incl snippet for cpp

### PR DESCRIPTION
**Pull request by tapanprakasht**
I have created  incl snippet for cpp which enables to include header files easily.

### CODE REVIEW

**Comment by lpil** [comment]
Is this general enough? Wouldn't people want to incl other things and use other name spaces?

**Comment by tapanprakasht** [comment]
Added uname snippet for adding namespaces.

File: snippets/cpp.snippets [hunk 1]
```diff
@@ -129,3 +129,6 @@ snippet lld
 	[${1}](${2}){
 		${3}
 	}
+snippet uname
```
Comment (SirVer) [inline snippets/cpp.snippets]: Using a namespace is relatively short, I do not think it needs a snippet. Making one is here
See https://github.com/honza/vim-snippets/blob/master/UltiSnips/cpp.snippets#L27.

Should use the same trigger, the UltiSnips one is much superior though.

**Comment by SirVer** [comment]
ping?

**Comment by SirVer** [comment]
closing for no reply.

**Pull request closed without merging**